sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -4550,9 +4562,31 @@ public void addCommand(Integer commandID,
 						   String menuText, Integer parentID, Integer position,
 						   Vector<String> vrCommands, String IconValue, ImageType IconType, Integer correlationID)
 			throws SdlException {
-		
-		AddCommand msg = RPCRequestFactory.buildAddCommand(commandID, menuText, parentID, position,
-			vrCommands, IconValue, IconType, correlationID);
+
+
+		AddCommand msg = new AddCommand(commandID);
+		msg.setCorrelationID(correlationID);

```

Comment:
we don't need to explicitly set correlation IDs for RPCs anymore, unless you have a reason to specifically track it for the response. https://github.com/smartdevicelink/sdl_android/blob/master/sdl_android/src/main/java/com/smartdevicelink/proxy/RPCRequest.java#L26-L28